trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.ts

```diff
@@ -6,6 +6,9 @@ import { PRIMARY_NAVIGATION_KEYS } from '../../../constants';
 import { mapFeeRecordItemsToFeeRecordViewModelItems } from '../helpers';
 import { UtilisationReportReconciliationForReportViewModel } from '../../../types/view-models';
 import { getAndClearAddPaymentFieldsFromRedirectSessionData } from './get-and-clear-add-payment-fields-from-redirect-session-data';
+import { FeeRecordItem } from '../../../api-response-types';
+
+const isAtLeastOnePaymentReceivedNonNull = (feeRecords: FeeRecordItem[]): boolean => feeRecords.some(({ paymentsReceived }) => paymentsReceived !== null);
```

Comment:
Nitpick but can we name this something like:
`feeRecordsHaveAtLeastOnePaymentReceived`

The current name make it sounds a bit like paymentsReceived can be null and that is a payment received rather than us just checking one exists (by checking it's not null)

Reply:
I'm not sure if I'm misunderstanding what you mean, but each `paymentsReceived` is of type `CurrencyAndAmount | null`, so it can be `null`. Either way, I do think updating the function name makes sense, but the check for it existing would be `!== undefined`, so the choice of `is<...>NonNull` was intentional